src/ucp/api/ucp.h

```diff
@@ -644,6 +690,41 @@ typedef struct ucp_worker_params {
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Tuning parameters for the UCP listener.
+ *
+ * The structure defines the parameters that are used for the
```

Comment:
This structure defines parameters for @ref ucp_worker_listen, which is used to listen for incoming client/server connections.